packages/ckeditor5-heading/theme/heading.css

```diff
@@ -18,3 +18,7 @@
 .ck[class*="ck-heading_heading"] {
 	font-weight: bold;
 }
+
+.ck.ck-list__item > .ck-button[class*="ck-heading_heading"] .ck-button__label {
+	line-height: var(--ck-font-size-large);
```

Comment:
I'm not so sure about this change. It creates a visual regression (see screenshots below)

**Current stable** https://ckeditor.com/docs/ckeditor5/latest/examples/builds/document-editor.html 

<img width="446" alt="image" src="https://github.com/ckeditor/ckeditor5/assets/1099479/6998d4ef-4af1-4be4-b595-b5120b8565a7">

**This PR**

<img width="783" alt="image" src="https://github.com/ckeditor/ckeditor5/assets/1099479/742ee650-f5ee-45f4-8b67-89ecf3b9f95d">

---

AFAIR this is customized on purpose by a dedicated CSS on our website

<img width="1919" alt="Screenshot 2024-06-19 at 14 49 33" src="https://github.com/ckeditor/ckeditor5/assets/1099479/1ca4a97f-71e1-4d12-afc6-4d2ee0ac0410">

Reply:
Yep, but on the other hand, it fixes overriding `line-height` CSS in new selector.  Without such rule it looks like this:

![obraz](https://github.com/ckeditor/ckeditor5/assets/3949262/0d430449-a367-428c-8d4b-c1624288e7bc)

Reply:
I managed to fix this in this [commit](https://github.com/ckeditor/ckeditor5/pull/16409/commits/f248f9f77fda933ef4abf5361e41297ff0d4780e#diff-89c661e74afa6f00cb58fc5bcd651f40c1750417127eae21b1d44d0f94ebf722R46). It should work properly.